code/game/objects/machinery/vending/marine_vending.dm

```diff
@@ -51,6 +51,7 @@
 		/obj/item/weapon/gun/launcher/m81 = 15,
 		/obj/item/explosive/grenade/frag = 30,
 		/obj/item/attachable/bayonetknife = 20,
+		/obj/item/clothing/suit/storage/marine/harness/boomvest = 20,
 		/obj/item/weapon/throwing_knife = 5,
```

Comment:
I'd rather they be Ops vendor rather than standard if you dont mind

Reply:
done